nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -168,7 +168,104 @@ INTERNAL_IPS = ("127.0.0.1", "::1")
+        "verbose": {
+            "format": "%(asctime)s.%(msecs)03d %(levelname)-7s %(name)-20s %(filename)-15s %(funcName)30s() :\n  %(message)s",
+            "datefmt": "%H:%M:%S",
+        },
+    },
+    "handlers": {},
+    "loggers": {},
+    "root": {
+        "level": LOG_LEVEL,
+    },
+}
+
+if os.environ.get("NAUTOBOT_LOGGING_PRESET", None) == "files":
+    LOGGING["handlers"] = {
+        "worker_log": {
+            "level": "DEBUG",
+            "class": "logging.handlers.TimedRotatingFileHandler",
+            "filename": "/var/log/nautobot/nautbot_worker.log",
+            "when": "midnight",
+            "utc": True,
+            "interval": 1,
+            "backupCount": 5,
+            "formatter": "verbose" if DEBUG else "normal",
+        },
+        "nautbot_log": {
+            "level": "DEBUG",
+            "class": "logging.handlers.TimedRotatingFileHandler",
+            "filename": "/var/log/nautobot/nautbot.log",
+            "when": "midnight",
+            "utc": True,
+            "interval": 1,
+            "backupCount": 5,
+            "formatter": "verbose" if DEBUG else "normal",
+        },
+        "nautbot_auth_log": {
+            "level": "DEBUG",
+            "class": "logging.handlers.TimedRotatingFileHandler",
+            "filename": "/var/log/nautobot/nautbot_auth.log",
+            "when": "midnight",
+            "utc": True,
+            "interval": 1,
+            "backupCount": 5,
+            "formatter": "verbose" if DEBUG else "normal",
+        },
+    }
+    LOGGING["loggers"] = {
+        "django": {"handlers": ["nautbot_log"], "level": DJANGO_LOG_LEVEL},
+        "rq.worker": {
+            "handlers": ["worker_log"],
+            "level": LOG_LEVEL,
+            "propagate": False,
+        },
+        "django_auth_ldap": {
+            "handlers": ["nautbot_auth_log"],
+            "level": LOG_LEVEL,
+            "propagate": False,
+        },
+        "nautbot.auth": {
+            "handlers": ["nautbot_auth_log"],
+            "level": LOG_LEVEL,
+            "propagate": False,
+        },
+    }
+    LOGGING["root"]["handlers"] = ["nautbot_log"]
+elif os.environ.get("NAUTOBOT_LOGGING_PRESET", None) == "console":
+    LOGGING["handlers"]["console"] = {
+        "level": "DEBUG",
+        "class": "rq.utils.ColorizingStreamHandler",
+        "formatter": "verbose" if DEBUG else "normal",
+    }
+    LOGGING["loggers"] = {
+        "django": {"handlers": ["console"], "level": DJANGO_LOG_LEVEL},
+    }
+    LOGGING["root"]["handlers"] = ["console"]
+elif os.environ.get("NAUTOBOT_LOGGING_PRESET", None) == "journal":
+    LOGGING["handlers"]["systemd"] = {
+        "level": "DEBUG",
+        "class": "systemd.journal.JournaldLogHandler",
+        "formatter": "verbose" if DEBUG else "normal",
+    }
+    LOGGING["loggers"] = {
+        "django": {"handlers": ["systemd"], "level": DJANGO_LOG_LEVEL},
+    }
+    LOGGING["root"]["handlers"] = ["systemd"]
+else:
+    LOGGING = {}
```

Comment:
IMO just ditch the logic here and define all of these handlers by default. Then just replace the logic with a bit of user-facing documentation, e.g. "If you want to log to a file use this handler, or log to console use this one. If you want to change the path to the log file in the default `nautobot_log` handler, change the `filename` variable."

For example, imagine something in the docs like "To enable debug logging to using the `nautobot_log` handler and change the file location in your `nautobot_config.py`":

```
LOGGING["loggers]["django"] = {"handlers": ["nautobot_log"], level="DEBUG"}
LOGGING["handlers"]["nautobot_log"]["filename"] = "/path/to/nautobot.log"

Reply:
Ok I removed the logic for simply a preset to configure logging for you and moved it all to the docs.  I set the basic configuration in `settings.py` so if people want to override the whole `LOGGING` config they can do that easily in the config